lib/host.py

```diff
@@ -328,7 +324,7 @@ def import_iso(self, uri, sr: SR):
 
         download_path = None
         try:
-            params = {'uuid': vdi_uuid}
+            params: Dict[str, Union[str, bool]] = {'uuid': vdi_uuid}
```

Comment:
That one feels strange, as we're only using `str` values, but that seems to be required because `xe()` wants that type.  It looks like that constraint only makes sense since `xe()` can in theory modify its `params` argument, and that the constraint would go away if we could annotate `params` as input-only argument, but I can't find if we can do that.

Reply:
It's indeed because `xe` `args` parameter is broken: https://github.com/xcp-ng/xcp-ng-tests/pull/292#issuecomment-2778002055
It's not really in the scope of the PR indeed, I will remove it.

Reply:
I can't remove it without making pyright unhappy